source/developers/concepts/hooks_extension_framework.rst

```diff
@@ -130,7 +130,34 @@ In summary, filters can be used when implementing application flow control that
 
 You can review the :doc:`Open edX Filters <openedx-filters:index>` documentation for more information on :doc:`openedx-filters:how-tos/using-filters` in your project. This documentation includes a list of :doc:`openedx-filters:reference/filters` and much more.
 
+Still Deciding Which to Use?
+----------------------------
+
+If you're still unsure whether to use an Open edX Event or Filter, ask yourself the following questions:
+
+**Does it change the default platform behavior?**
+
+- **Yes:** For example, the course enrollment process now depends on a third-party subscription service. This modifies the default enrollment process.
+- **No:** For example, when generating certificates, you may need to create credentials in an external service, but the default process remains unchanged.
+
+Filters are useful when you need an immediate response that directly modifies the caller process and impacts the rest of the flow. In contrast, events are more decoupled from the caller process. They do not return a response, leaving the caller process unchanged.
+
+**If the answer yes, should it be a filter? Does the application behavior benefit from being altered?**
+
+- If so, a filter may suit your needs.
+- If not, maybe because your use case should be the default offering of the platform, contribute changes if they benefit the community, but check with the `Product Working Group`_ first.
```

Comment:
I might say
```suggestion
- If not, maybe because your use case should be the default offering of the platform, contribute changes if they benefit the community, consider submitting a `Product Proposal`_ to begin discussing changes to the default offering.
```

and then link to https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review

Reply:
Thank you! https://github.com/openedx/docs.openedx.org/pull/618/commits/b253115eb7b78c76a9664d100c5e6ec06372b18b